src/iso19111/io.cpp

```diff
@@ -9465,7 +9468,16 @@ PROJStringParser::createFromPROJString(const std::string &projString) {
                         iFirstAxisSwap < iFirstGeogStep ? iSecondAxisSwap
                                                         : iFirstAxisSwap));
                 if (iter == 1) {
-                    return nn_static_pointer_cast<BaseObject>(obj);
+                    if (conversionStructure) {
+                        auto projObjCrs =
+                            nn_dynamic_pointer_cast<ProjectedCRS>(obj);
+                        if (projObjCrs) {
+                            return projObjCrs->derivingConversion();
```

Comment:
to avoid the added pipeline / unit conversion, which is linked to derivingConversion() still linking to a base geodetic CRS, you'd probably need to do projObjCrs->derivingConversion()->shallowCloneNoCRS() that would create a 'standalone' conversion with no links to the source geodetic CRS and target projected CRS

with a new method shallowCloneNoCRS(), being a variant of the existing shallowClone() that doesn't set the source&target CRS:

ConversionNNPtr Conversion::shallowCloneNoCRS() const {
    auto conv = Conversion::nn_make_shared<Conversion>(*this);
    conv->assignSelf(conv);
    return conv;
}

Reply:
That didn't seem to work. I am wondering if the `ellps=WGS84` added breaks things.